src/js/ui/menu.js

```diff
@@ -308,7 +308,7 @@ Menu.prototype.item = function(sectionIndex, itemIndex, item) {
 
 Menu.prototype.selection = function(callback_or_sectionIndex, itemIndex) {
   if (typeof callback_or_sectionIndex === 'function') {
-    this._selections.push(callback);
+    this._selections.push(callback_or_sectionIndex);
```

Comment:
`this._selections` would still have to be populated though.

```javascript
Menu.prototype.selection = function(sectionIndex, itemIndex) {
  var callback;
  if (typeof sectionIndex === 'function') {
    callback = sectionIndex;
    sectionIndex = undefined;
    this._selections.push(callback);
  } 
  this._selection = {
    sectionIndex: sectionIndex,
    itemIndex: itemIndex
  };
  this._select();
};
```

Otherwise the callback would never be fired, would it?